### PR TITLE
{lang}[intel/2015b] Perl 5.20.2-bare + 5.22.0-bare (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c8fdabef3fa020479224 for a full test report.

**Comment by boegel** [comment]
@wpoely86, @gjbex, @backelj, @hajgato: please test?

File: easybuild/easyconfigs/p/Perl/Perl-5.22.0-intel-2015b-bare.eb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = 'Perl'
+version = '5.22.0'
+versionsuffix = '-bare'
+
+homepage = 'http://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Perl/Perl-5.22.0-intel-2015b-bare.eb]: a or b? Check PR title.

Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.22.0-intel-2015b-bare.eb]: oh, woops. thanks

Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.22.0-intel-2015b-bare.eb]: Jenkins will trip over this too though, but nice catch!

**Comment by backelj** [comment]
Test report by @backelj
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/e45631324bfdd710af06 for a full test report.

**Comment by boegel** [comment]
@backelj: you only built one of both?

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/52844bce0f8abdcdee7e for a full test report.

**Comment by backelj** [comment]
Test report by @backelj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/ed9f0e12eb25c83e3846 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/298f071ea5cf44d5c371 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/168df51d80c0039db63e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bd122104214595dc979b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3509b42f318c1121c86d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e099f9d5168d2b1a483f for a full test report.

**Comment by boegel** [comment]
hmm, I'll need to figure out why this occurred on the RHEL6 system in Leuven:

```
ERROR Not all FFTW interface libraries ['fftw3xc_intel_pic', 'mkl_intel_lp64', 'mkl_sequential', 'mkl_core'] are found in ['/scratch/leuven/400/vsc40023/software/imkl/11.2.3.187-iimpi-7.3.5-GNU-4.9.3-2.25/mkl/lib/intel64', '/scratch/leuven/400/vsc40023/software/imkl/11.2.3.187-iimpi-7.3.5-GNU-4.9.3-2.25/compiler/lib/intel64'], can't set FFT_LIB. (at easybuild/toolchains/fft/intelfftw.py:94 in _set_fftw_variables)
```

@gjbex: have you seen this occurring on the @hpcleuven systems?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by fgeorgatos** [comment]
watch out also with spurious licensing issues:
https://github.com/hpcugent/easybuild-easyblocks/issues/103

other things to try: builds on local/ramdisk filesystems, compare hashes of sourcefiles, force rebuilds etc.

**Comment by boegel** [comment]
@fgeorgatos: it's probably the overly active cleanup script they have running on the `scratch` filesystem causing this; installing `HPL` on top of `intel/2015b` worked fine before (and the error is caused by the EasyBuild toolchain mechanism)

I'll retry after reinstalling on a different filesystem that doesn't get cleaned up automagically.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.6.6
See https://gist.github.com/bc0d55c3ce42b7830c7c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/46e919eaf2640fcac4b9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f4cd4348eb7e95ba94f1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/90840736e9688eb6fbd2 for a full test report.

**Comment by boegel** [comment]
Good to go after extensive testing, thanks to everyone who contributed to this!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5a3677449c664145a09b for a full test report.

**Comment by backelj** [comment]
Test report by @backelj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/bab909e9841869473d3a for a full test report.

**Pull request merged**